### PR TITLE
Init basic e2e tests

### PR DESCRIPTION
**Pull request by balzdur**
I stop here for the first iteration.

It would be great when you have time to test on your side if you struggle to run those tests

### CODE REVIEW

File: packages/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+> **Note:** The tests in this package are designed to be run against a local test environment. Steps 1 and 2 are manual steps that will be automated in the future.
+
+1. Create the test organisation (if it does not already exist):
+   1. Open the backoffice application
+   2. Create a new organisation with the name `e2e`. Do not forget to check "init with demo data"
+   3. Create a new admin user on this org with the email `admin@e2e.com`
+2. Start a local test environment:
+   1. start the Firebase emulator suite
+   2. start the backend normally
+   3. start the front-end application normally
```
Comment (Pascal-Delange) [inline packages/tests/README.md:20]: I first had to:
- go to packages/tests
- run `pnpm exec playwright install`
Now the tests are running but in error

I see a bunch of requests received by the backend 
Capture d’écran 2024-09-05 à 21 47 22

but then the tests times out 

```1) [chromium] › 001-user-signin-flow.test.ts:16:1 › Users can sign in with Google ────────────────

    Tearing down "firebase" exceeded the test timeout of 30000ms.

    Error: locator.click: Test ended.
    Call log:
      - waiting for getByLabel('Open menu for user jbe@zorg.com')


       at ../page-object-models/firebase-auth-emulator.ts:45

      43 |   async deleteUser(email: string) {
      44 |     await this.newPage(async (page) => {
    > 45 |       await page.getByLabel(`Open menu for user ${email}`).click();
         |                                                            ^
      46 |       await page.getByRole('menuitem', { name: 'Delete user' }).click();
      47 |       await page.getByRole('button', { name: 'Delete' }).click();
      48 |       await expect(

        at cb (/Users/pascal/Documents/marble/marble-frontend/packages/tests/page-object-models/firebase-auth-emulator.ts:45:60)
        at FirebaseAuthEmulatorPage.newPage (/Users/pascal/Documents/marble/marble-frontend/packages/tests/page-object-models/firebase-auth-emulator.ts:13:11)
        at FirebaseAuthEmulatorPage.deleteUser (/Users/pascal/Documents/marble/marble-frontend/packages/tests/page-object-models/firebase-auth-emulator.ts:44:5)
        at Object.firebase (/Users/pascal/Documents/marble/marble-frontend/packages/tests/fixtures/firebase.ts:49:7)
```

Comment (Pascal-Delange) [reply packages/tests/README.md:20]: Is there a particular project id that I need to run the firebase emulator with ?

Comment (balzdur) [reply packages/tests/README.md:20]: This is good, it answer one of my concerns: prevent `pnpm install` to be bloated by playwright "full" install. So doing a manual step feels like a good thing right now, I'll add it to the Readme.

Concerning the FB emulator, I use the same one we use for dev, but it shouldn be a problem (using `firebase emulators:start` from the backend working directory works on my side)

Are you up to date with origin ? Reading the error message, it tries to delete `jbe@zorg.com` but now, tests are relying on a separate user `admin@e2e.com` so it's weird.

**Review by Pascal-Delange — APPROVED** [review]
I was able to run the tests successfully, with and without ui !
Looks all good :-)

I had a weird thing where firebase was coming from my /opt/homebrew/bin folder, even though I didn't install it with brew (at least I didn't have it in my brew kegs - possibly it was from an old installation). Anyway, I removed it on homebrew and installed it with npm.

Let's please just put a note in the readme (and possibly in our internal notion - frontend wiki ?) that explains why we have to use a specific version of the firebase cli, and which version is supported.

**Comment by balzdur** [comment]
I updated :
- the readme, to point to the already updated doc in the backend readme (the one in charge of explaining how to install + run the emulator)
- the frontend wiki
  - update on some existing doc (not linked to this PR scope but it's done)
  - creation of a test basic page, to reference our different testing strategies and the "why" we need to use a specific FB emulator version

**Pull request merged**